README.rst

```diff
@@ -40,13 +40,29 @@ Development instructions
 
 Initial setup and installation:
 
-- recommended: create and activate a python 3.5 virtualenv::
+- recommended: create and activate a python 3.8 virtual environment. Using pyenv:
 
-    virtualenv mep -p python3.5
-    source mep/bin/activate
+    # if pyenv is not installed
+    curl https://pyenv.run | bash
+
+    # install 3.8
+    pyenv install 3.8.14
+
+    # activate 3.8
+    pyenv shell 3.8.14    
+
+    # create virtual environmnt
+    python -m venv venv
+
+    # activate virtual environment
+    . venv/bin/activate
 
 - Install required python dependencies::
 
+    # make sure latest pip and wheel package is installed
+    pip install -U pip wheel
+
+    # install requirements
     pip install -r requirements.txt
     pip install -r dev-requirements.txt
```

Comment:
wheel is included dev requirements, what if we just switch the order of instructions here? is that sufficient?

Reply:
That should work!